src/utils/config_filters.cpp

```diff
@@ -121,3 +121,144 @@ bool utils::config_filters::bool_or_empty(const config& filter, const config& cf
 	}
 	return is_matches;
 }
+
+bool utils::config_filters::matches_ability_filter(const config & cfg, const std::string& tag_name, const config & filter)
```

Comment:
I don't know why the function was moved here. I don't think it belongs here.

Reply:
moved to abilities.cpp